packages/fxa-settings/tailwind.config.js

```diff
@@ -177,6 +188,11 @@ module.exports = {
         900: '#321C64',
       },
     },
+    flex: {
+      1: '1',
+      2: '2',
+      4: '4',
+    },
```

Comment:
Should this live inside the `extend` object?

Reply:
There's only one `flex-#` (`flex-1`) class and I didn't realize that class was even there, so I'll move this under `extend` with only 2/4 needed 👍